Kernel/Config/Defaults.pm

```diff
@@ -2401,7 +2401,10 @@ sub AutoloadPerlPackages {
                 my $FileName = $Package =~ s{::}{/}smxgr;
 
                 require $FileName . '.pm'; ## nofilter(TidyAll::Plugin::OTOBO::Perl::Require)
-            };
+            }
```

Comment:
Looks fine. No need to check the return value of require as require throws an exception in case of problems.